### PR TITLE
[APB-1227][AO,RV] add test-only endpoint for stride auth

### PR DESCRIPTION
**Pull request by rvijayan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,27 @@
...
+ - Agent who needs an Agent PAYE reference code is able to enter its details and can be issued a code
+ 
+ ## Running the tests
+ 
+     sbt test it:test
+ 
+ ## Running the app locally
+ 
+     sm --start MONGO   
+     sm --start AGENT_EPAYE_REG -f
```
Comment (johnrevill) [inline README.md]: AGENT_EPAYE_REGISTRATION ?

Comment (johnrevill) [reply README.md]: Ah, just seen the profile, sorry. Ignore that comment

File: app/ErrorHandler.scala [hunk 2]
```diff
@@ -38,9 +41,14 @@ class ErrorHandler @Inject()(implicit configuration: Configuration, val messages
   }
 
   override def onServerError(request: RequestHeader, exception: Throwable): Future[Result] = {
-    Future successful InternalServerError(error_template(
-      Messages("global.error.500.title"),
-      Messages("global.error.500.heading"),
-      Messages("global.error.500.message"))).withHeaders(CACHE_CONTROL -> "no-cache")
+    val response = exception match {
```
Comment (ZDevelop94) [inline app/ErrorHandler.scala]: could remove response variable and just do Future.successful(exception... to clean up code

Comment (rvijayan) [reply app/ErrorHandler.scala]: IMHO, Having a Future.successful {} for exception matcher might make the code bit unreadable. I know it's just a matter of style :)

**Pull request merged**